src/org/usfirst/frc/team1922/robot/commands/TurnTo_Command.java

```diff
@@ -25,20 +27,40 @@ public TurnTo_Command(int direction) {
     @Override
     protected void initialize() {
         start = Robot.m_driveTrain.getAngle();
-        Robot.m_driveTrain.drive(direction*(.2), -direction*(.2));
+        if( direction >= 0 ){
```

Comment:
This chunk of code can be made more readable.

Firstly the drive speed should be the same between both directions. That's just different because of the experiment we were running though. Secondly though let's put that in a variable somewhere. That will make it very clear what's going on.

Finally there's this strange asymmetry where `isLeft` is set in one branch but not the other. It works because we set the default value when we declared the variable but it still makes it difficult to read. When I'm reading this chunk of code I can't really know what isLeft is going to be set to if direction is >= 0. 

We can do better by directly assigning isLeft to the thing we want and then writing our if condition based on that. 
```
isLeft = direction < 0;  //if the direction passed in is < 0 we should turn left
if(isLeft)
{
    Robot.m_driveTrain.drive(-1 * turnSpeed, turnSpeed);
}
else
{
    Robot.m_driveTrain.drive(turnSpeed, -1 * turnSpeed)
}
```